doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,187 @@
+
+
+Optional configuration
+-----------------------
+
+You can also increase the UART throughput by changing the baudrate.
+Some of the trace logs will be dropped if the throughput is too low.
+By default, the UART baudrate is set to ``115200``.
+To increase the baudrate to ``1000000``, add the ``current-speed = <1000000>;`` property to the ``uart1`` node in the DTS overlay file.
+This can be done like the following:
+
+.. code-block:: devicetree
+
+      &uart1 {
+         current-speed = <1000000>;
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+
+.. _ug_zigbee_configuring_zboss_traces_using_usb:
+
+Trace logs using native USB
+===========================
+
+Trace logs can also be configured to use a native USB.
+This is useful because trace logs will be printed through a separate virtual COM port so that the console logs can still be read through the JLink COM port.
+For applications that relay on the UART connection through the JLink COM port, for example the Network co-processor (NCP) sample, trace logs can only be configured through USB (COM port emulated over USB).
+See the :ref:`zigbee_ncp_sample` sample page for how to configure trace logs for USB in this case.
+
+.. note::
+   Before proceeding with the following steps, first check if your Zigbee application already has USB enabled or is currently using a USB.
+   If your application is already using a virtual COM port via native USB, use a device name that is different than the default ``CDC_ACM_0`` to create new virtual COM port for printing trace logs.
+   For example, if ``CDC_ACM_0`` is already present, then create a virtual COM port named ``CDC_ACM_1``, and so on.
+   Additionally, the Kconfig option :kconfig:`CONFIG_USB_COMPOSITE_DEVICE` must be set if there are multiple virtual COM ports configured.
+
+   See the :ref:`zigbee_ncp_sample` sample page as an example where one virtual COM port instance is already configured, and another must be created.
+
+To configure trace logs using native USB, complete the following steps:
+
+1. Set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING`.
+   This also enables the necessary USB Kconfig options.
+
+#. Configure which USB device to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   This is optional, as the default ``CDC_ACM_0`` will be used if no other USB device is configured.
+
+#. Create a virtual COM port that will be used for printing ZBOSS trace logs by extending the DTS overlay file for the selected board with the following:
+
+   .. code-block:: devicetree
+
+      &zephyr_udc0 {
+         cdc_acm_uart0: cdc_acm_uart0 {
+            compatible = "zephyr,cdc-acm-uart";
+            label = "CDC_ACM_0";
+         };
+      };
+
+   .. note::
+      For the ZBOSS trace logs to be printed correctly through the USB, it is recommended to not use the USB autosuspend.
```

Comment:
```suggestion
      For the ZBOSS trace logs to be printed correctly through the USB, it is recommended to avoid using the USB autosuspend.
```
Somehow I could not see `not` clearly there when I read the preview...